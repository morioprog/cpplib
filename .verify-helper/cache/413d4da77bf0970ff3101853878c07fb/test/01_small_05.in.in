9 8
1 2
3 4
5 6
7 8
0 2
0 3
0 6
0 7
