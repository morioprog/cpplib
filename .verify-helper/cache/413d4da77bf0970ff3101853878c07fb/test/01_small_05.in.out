5
1
0
7
8
6
3
4
2
